app/cypress/integration/cif/project-revision-change-logs.spec.js

```diff
@@ -41,5 +41,7 @@ describe("the project amendment and revisions page", () => {
     cy.happoAndAxe("Amendment & Revision View", "view", "main", true);
     cy.findByRole("link", { name: /view minor revision 1/i }).should("exist");
     cy.get('input[value="Minor Revision"]').should("be.checked");
+    cy.get('[aria-label="Status"]').contains("Applied");
```

Comment:
The card calls this "Approved" but I believe it was decided that this would be renamed to "Applied" throughout?